packages/react-server/core/ClientController.js

```diff
@@ -69,7 +76,6 @@ class ClientController extends EventEmitter {
 		ReactServerAgent.cache().rehydrate(dehydratedState.InitialContext['ReactServerAgent.cache']);
 		this.mountNode = document.getElementById('content');
 
-		this._setupFramebackController();
```

Comment:
This is attached to the request context now, since the navigator needs to get at it.